docs/source-stepping.md

```diff
@@ -0,0 +1,9 @@
+# Source Stepping
+
+In most cases, you'll never need to step through the FakeItEasy source code, but occasionally, it may help to see what FakeItEasy is doing under the hood. FakeItEasy uses [GitLink](https://github.com/GitTools/GitLink) to make it a breeze to step into the FakeItEasy source code.
+
+To enable GitLink to work in Visual Studio:
+
+1. Tools -> Options -> Debugging -> General -> Enable Just My Code: **Uncheck**
+1. Tools -> Options -> Debugging -> General -> Enable source server support: **Check**
+1. Tools -> Options -> Debugging -> Symbols -> Cash symbols in this directory: **Path to a suitable folder**

```

Comment:
I'm surprised you're a "->" guy and not a "→" guy.

Also "Cache" instead of "Cash".
Or is there a finder's fee I don't know about?